### PR TITLE
OC-9834: (Updated) Checking for existing participants Id without case sensitive while creating participants

### PR DESCRIPTION
**Pull request by suthirr-openclinica**
updated as per the comment in the code review

### CODE REVIEW

File: web/src/main/java/org/akaza/openclinica/controller/AccountController.java [hunk 1]
```diff
@@ -24,6 +24,7 @@
 import org.akaza.openclinica.i18n.util.ResourceBundleProvider;
 import org.akaza.openclinica.service.pmanage.ParticipantPortalRegistrar;
 import org.apache.commons.dbcp2.BasicDataSource;
+import org.apache.xpath.operations.Bool;
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/controller/AccountController.java]: remove unnecessary import

File: web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java [hunk 2]
```diff
@@ -122,6 +122,7 @@
 					+ "<br />phoneNumberTooLong                            : Phone number length should not exceed 15 characters."
 					+ "<br />invalidPhoneNumber                            : Phone number should not contain alphabetic characters."
 					+ "<br />participateModuleNotActive                    : Participant Module is Not Active."
+					+ "<br />participantIDAlreadyExists					   : Participant id is already exists."
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java]: remove the extra "is" and capitalize ID so it reads - "Participant ID already exists."

File: web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java [hunk 3]
```diff
@@ -177,6 +178,7 @@
...
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java]: remove the extra "is" and capitalize ID so it reads - "Participant ID already exists."

File: web/src/main/java/org/akaza/openclinica/service/ParticipantServiceImpl.java [hunk 4]
```diff
@@ -115,8 +115,13 @@ public String createParticipant(SubjectTransferBean subjectTransfer,StudyBean cu
         // create subject
         StudyBean siteStudy = subjectTransfer.getSiteStudy();
         StudySubject studySubject=null;
-        StudySubjectBean studySubjectBean  =getStudySubjectDao().findByLabelAndStudy(subjectTransfer.getPersonId(), currentStudy);
 
+        StudySubjectBean studySubjectBean  =getStudySubjectDao().findByLabelAndStudyForCreatingParticipant(subjectTransfer.getPersonId(), currentStudy.getId());
+
+        StudySubjectBean studySubjectBeanInParent = new StudySubjectBean();
+        if (currentStudy.getParentStudyId() > 0) {
+            studySubjectBeanInParent = getStudySubjectDao().findByLabelAndStudyForCreatingParticipant(subjectTransfer.getPersonId(), currentStudy.getParentStudyId());// <
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/service/ParticipantServiceImpl.java]: remove the comment at the end

File: web/src/main/java/org/akaza/openclinica/controller/AccountController.java (hunk 1 above)
Comment (suthirr-openclinica) [reply web/src/main/java/org/akaza/openclinica/controller/AccountController.java]: Removed the imports

File: web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java (hunk 2 above)
Comment (suthirr-openclinica) [reply web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java]: Changed as per the comments

File: web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java (hunk 3 above)
Comment (suthirr-openclinica) [reply web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java]: Changed as per the comments

File: web/src/main/java/org/akaza/openclinica/service/ParticipantServiceImpl.java (hunk 4 above)
Comment (suthirr-openclinica) [reply web/src/main/java/org/akaza/openclinica/service/ParticipantServiceImpl.java]: Changed as per the comments

**Pull request closed without merging**